sos/plugins/insights.py

```diff
@@ -13,16 +13,29 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
+<<<<<<< HEAD
```

Comment:
Are you sure you want this line in your commit??

Reply:
You need to `git rebase -i`, and then use squash or fixup commands to combine all of these commits into a single commit that applies to `master`:

```
commit 9e5d619e89b67bdf412e96715893a6139904f1ab (HEAD -> bmr-tmp)
Author: Ashish Humbe <ahumbe@redhat.com>
Date:   Fri Oct 5 14:00:28 2018 +0530

    [insights] update config and log file

commit 6c134929569d892d01ce68ae02cb1810eee4f681
Author: Ashish Humbe <ahumbe@redhat.com>
Date:   Fri Oct 5 13:21:32 2018 +0530

    [insights] update config and log file

commit 2ca7cbcd4c1b0f617e6ff2200a0281e172e9eba7
Merge: 939d32eb 1f5570a5
Author: Ashish Humbe <ahumbe@redhat.com>
Date:   Fri Oct 5 13:13:29 2018 +0530

    Merge branch 'insights_log' of https://github.com/ahumbe/sos into insights_log

commit 939d32eb9c23ab6aaaa62ec34dc8d0e7449388f4
Author: Ashish Humbe <ahumbe@redhat.com>
Date:   Wed Sep 12 15:41:13 2018 +0530

    [insights] update config and log file

commit 1f5570a5afa3aa0dfc6590e275ba83d3321e3a13
Author: Ashish Humbe <ahumbe@redhat.com>
Date:   Wed Sep 12 15:41:13 2018 +0530

    [insights] update config and log file

```